src/ray/core_worker/lib/java/org_ray_runtime_WorkerContext.cc

```diff
@@ -0,0 +1,129 @@
+#include "ray/core_worker/lib/java/org_ray_runtime_WorkerContext.h"
+#include <jni.h>
+#include "ray/common/id.h"
+#include "ray/core_worker/context.h"
+#include "ray/core_worker/lib/java/jni_helper.h"
+
+inline ray::WorkerContext *GetWorkerContext(jlong nativeWorkerContext) {
```

Comment:
`GetWorkerContextFromPointer()` will be more clear. Otherwise it looks like a getter.

Reply:
OK.